### ISSUE TITLE
Metadata link broken 10.3.0-rc2

### ISSUE THREAD

**Issue by grafixeyehero** [jellyfin/jellyfin#1231]
**Describe the bug**

**To Reproduce**

1. Go to 'Home Menu'
2. Click on 'Metadata'

**System (please complete the following information):**
 - OS: [Windows]
 - Browser: [Firefox, Chrome]
 - Jellyfin Version: [10.3.0-rc2]

**Comment by cesalo** [comment]
I just tested this and is correct, Metadata link doesn't do anything (I'm in Centos).

**Comment by joshuaboniface** [comment]
Looks like there's a bad path:

```
Error: Load failed: scripts/edititemmetadata: http://jellyfin/web/scripts/edititemmetadata.js?v=10.3.0
```
Looks like a case where the script was moved (to `web/controllers/edititemmetadata.js`) but the reference not updated. Should be an easy fix - PR incoming.

**Comment by joshuaboniface** [comment]
Should be fixed in 10.3.0 release.
